spec/models/conference_spec.rb

```diff
@@ -1656,4 +1656,32 @@
       expect{ conference.save }.to change{ EventSchedule.count }.from(2).to(1)
     end
   end
+
+  describe 'revision count' do
+
+    let(:track) { create(:track, program: subject.program) }
+    let(:event) { create(:event, program: subject.program, track: track) }
+    let(:venue) { create(:venue, conference: subject) }
+    let(:room)  { create(:room, venue: venue) }
+
+    context 'should increment' do
```

Comment:
Contexts are used when you have different cases, such as:

```ruby
context 'when user is logged in' do #....
context 'when user is logged out' do #....
```

As such, it does not make much sense to use one single `context`, so in this case we can just remove it.